### PR TITLE
Business features step: include Mailpoet

### LINKED ISSUE #6489: Business features step: include Mailpoet
**Description:** the experiment implemented in #5287 showed positive results. Recap here: pbIJXs-GS-p2. We need to start including Mailpoet in this flow, as requested here: pbIJXs-G8-p2

**Acceptance criteria:**
- [ ] Include Mailpoet in the list for any country within the "Grow your store" section: 
Copy: Level up your email marketing with [Mailpoet](https://www.mailpoet.com/woocommerce-email-marketing/)
wp.org link to download plugin: https://wordpress.org/plugins/mailpoet/
- [ ] Display extensions in the following order:
Mailpoet
Facebook
Google Ads
Mailchimp
Creative Mail
- [ ] Change the Creative Mail copy to: Emails made easy with Creative Mail

### PR DESCRIPTION
**Pull request by octaedro**
Fixes #6489

This is a follow-up to [PR 6503](https://github.com/woocommerce/woocommerce-admin/pull/6503).
Here the "MailPoet" option is added to the new Business Details step in the OBW.

### Accessibility

-   [ ] I've tested using only a keyboard (no mouse)
-   [x] I've tested using a screen reader
-   [x] All animations respect [`prefers-reduced-motion`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion)
-   [x] All text has [at least a 4.5 color contrast with its background](https://webaim.org/resources/contrastchecker/)

### Screenshots
screenshot-one wordpress test-2021 03 03-17_08_23

### Detailed test instructions:

- Create a brand new site and go to the OBW.
- In the first OBW step (`Store Details`) set `US` in the `Country / Region` selector.
- Set `Food and drink` in the second step (`Industry`)
- Continue with the profiler.
- In the 4th step (`Business Details`) choose any of the options in both selectors.
- Under `Free features` tab, verify that the displayed extensions are:
```
Mailpoet
Facebook
Google Ads
Mailchimp
Creative Mail
```
(In that order)
- Verify that the Creative Mail option copy is `Emails made easy with Creative Mail`.

### CODE REVIEW

File: TESTING-INSTRUCTIONS.md [hunk 1]
```diff
@@ -2,6 +2,24 @@
 
 ## Unreleased
 
+### Store profiler - Added MailPoet to new Business Details step  #6515
+
+- Create a brand new site and go to the OBW.
+- In the first OBW step (`Store Details`) set `US` in the `Country / Region` selector.
+- Set `Food and drink` in the second step (`Industry`)
```
Comment (louwie17) [inline TESTING-INSTRUCTIONS.md:9]: Given https://github.com/woocommerce/woocommerce-admin/pull/6508 is merged, this is not required anymore, as long as it is not the CBD option. Country list is expanded now as well, as mentioned in the other PR, but sticking with `US` here is fine.

**Review by louwie17 — APPROVED** [review]
Tested well and changes look good @octaedro 👍 I will approve this on the premise that you will fix the E2E tests :) 
I also left one small comment for the testing instructions.
The E2E test fix is changing [this ](https://github.com/woocommerce/woocommerce-admin/blob/main/tests/e2e/specs/activate-and-setup/complete-onboarding-wizard.test.js#L64) from `7` to `8`
And the default number from `8` to `9` [here](https://github.com/woocommerce/woocommerce-admin/blob/main/tests/e2e/specs/activate-and-setup/complete-industry-section.js#L18) (I think those are the only one's).

**Comment by octaedro** [comment]
Hi @louwie17,

Thank you for your review 🙂

> The E2E test fix is changing this from 7 to 8
And the default number from 8 to 9 here (I think those are the only one's).

I think that just modifying [this line](https://github.com/woocommerce/woocommerce-admin/pull/6515/commits/d97ba3ee7472676936da79ced4a956ba1e513f3d#diff-00fbad14779d472d62b8b3e838782a37ab3c73d69953f9cff45ccf14a88c5604R78) should be enough to fix the e2e tests. The 2 lines you mentioned are related to the `Industry` step in the OBW but this PR only modifies the `Business Details` step.
Please let me know if I'm missing something.

Could you give this PR another look?

File: TESTING-INSTRUCTIONS.md (hunk 1 above)
Comment (octaedro) [reply TESTING-INSTRUCTIONS.md]: Sounds good to me 🙂
This was changed in the commit b43b1a0

**Comment by louwie17** [comment]
@octaedro yeah looks like fixing that line was all that was needed, thanks 👍
It's good to merge 💯

**Pull request merged**